homeassistant/components/cover/mqtt.py

```diff
@@ -51,44 +77,70 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     value_template = config.get(CONF_VALUE_TEMPLATE)
     if value_template is not None:
         value_template.hass = hass
+    tilt_value_template = config.get(CONF_TILT_VALUE_TEMPLATE)
+    if tilt_value_template is not None:
+        tilt_value_template.hass = hass
     add_devices([MqttCover(
         hass,
         config.get(CONF_NAME),
         config.get(CONF_STATE_TOPIC),
         config.get(CONF_COMMAND_TOPIC),
+        config.get(CONF_TILT_STATE_TOPIC),
+        config.get(CONF_TILT_COMMAND_TOPIC),
         config.get(CONF_QOS),
         config.get(CONF_RETAIN),
         config.get(CONF_STATE_OPEN),
         config.get(CONF_STATE_CLOSED),
         config.get(CONF_PAYLOAD_OPEN),
         config.get(CONF_PAYLOAD_CLOSE),
         config.get(CONF_PAYLOAD_STOP),
+        config.get(CONF_TILT_STATE_OPEN),
+        config.get(CONF_TILT_STATE_CLOSED),
+        config.get(CONF_TILT_PAYLOAD_OPEN),
+        config.get(CONF_TILT_PAYLOAD_CLOSE),
+        config.get(CONF_TILT_PAYLOAD_STOP),
         config.get(CONF_OPTIMISTIC),
+        config.get(CONF_TILT_OPTIMISTIC),
         value_template,
+        tilt_value_template,
     )])
 
 
 class MqttCover(CoverDevice):
     """Representation of a cover that can be controlled using MQTT."""
 
-    def __init__(self, hass, name, state_topic, command_topic, qos,
+    def __init__(self, hass, name, state_topic, command_topic,
+                 tilt_state_topic, tilt_command_topic, qos,
                  retain, state_open, state_closed, payload_open, payload_close,
-                 payload_stop, optimistic, value_template):
+                 payload_stop, tilt_state_open, tilt_state_closed,
+                 tilt_payload_open, tilt_payload_close, tilt_payload_stop,
+                 optimistic, tilt_optimistic, value_template,
+                 tvt):
         """Initialize the cover."""
         self._position = None
         self._state = None
+        self._tilt_position = None
+        self._tilt_state = None
```

Comment:
You are correct but I'm realizing that this _should_ be used just like `self._state` is.